### DISCUSSION TITLE
CAN Button commands/functions be checked in LightBlue app for Android.

### DISCUSSION THREAD

**Discussion by Celticpure** [alf45tar/PedalinoMini#402] [Q&A]
Hey everyone. This project is awesome so thank you. 

Can the assigned button commands, be it CC, PC etc be checked in the android app LightBlue?

The reason I ask, I like to have another way of checking that a particular command is sent via BLE. And LightBlue can check these upon the assigned button press etc. 

Hope that makes sense guys.

Thanks for your time.

**Comment by alf45tar** [comment]
Yes, you can but it is not recommended because you see only hex values.
If you need transcoded MIDI messages I suggest MIDI BLE Connect/MIDI Scope from Mobileer Inc. or MIDI Connector - RTP, USB, BLKE from Abraham Wisman.

**Reply by Celticpure** [reply]
Hex values are fine, it is really just to check the command is being sent from Pedalino. I am having trouble connecting to pedalino to read the sent command to LightBlue, LightBlue is saying it can't connect. But yet connects at the Bonded tab within the Lighthouse app. Just does not connect at the read hex page. 

Maybe I need to completely erase my devkit esp32 and reflash Pedalino. And try again.

**Reply by alf45tar** [reply]
I am using the first tab to connect. Not the Bonded.

**Reply by Celticpure** [reply]
![Screenshot_2022-11-24-08-12-16-656_com.appsforamps.katana.jpg](https://user-images.githubusercontent.com/96823448/203728514-e3ffa8f4-efd7-4160-a2e6-d22d5ac68b26.jpg)

Ok I can see it now reacting to the assigned CC command upon button press in LightBlue. 

However I want to use Pedalino to control my Boss Katana amp through the Android app Librarian. Librarian can see the Pedalino and connect but no commands are being received. Is there something I am missing here? See screenshot above.

**Reply by Celticpure** [reply]
Ok scratch that, got Pedalino working now with Katana Librarian. I had to clear previous BT connections a few times and now it connects fine and sends commands. GREAT!
